src/validation.h

```diff
@@ -666,7 +666,6 @@ class CChainState
     const std::unique_ptr<CChainstateHelper>& m_chain_helper;
     const std::unique_ptr<llmq::CChainLocksHandler>& m_clhandler;
     const std::unique_ptr<llmq::CInstantSendManager>& m_isman;
-    const std::unique_ptr<llmq::CQuorumBlockProcessor>& m_quorum_block_processor;
```

Comment:
it's possible to drop class forward declration also:

```diff
class CInstantSendManager;
-class CQuorumBlockProcessor;
} // namespace llmq 
```

Reply:
Resolved in latest push.